tests/test_fast_ion.py

```diff
@@ -65,15 +66,15 @@ def test_Gamma_c_Nemov_1D():
     Y_B = 100
     num_transit = 10
     num_well = 20 * num_transit
-    rho = np.linspace(0, 1, 10)
+    rho = np.linspace(1e-12, 1, 10)
```

Comment:
what changed at 0? or was it just analytically 0 before and you want to check it is not blowing up?

Reply:
It's zero analytically on axis, which I confirmed when I wrote the quantity. The reason for the change is here
- For the 2D methods, including zero will compute the axis limits of the dependencies, which will slow down the tests for no reason.
- For the 1D methods, `get_rtz_grid` makes a jitable grid by default which [moves the nodes to 1e-12](https://github.com/PlasmaControl/DESC/blob/6b069f3f2994de2f0297dce876b223eb8aedd6a1/desc/grid.py#L764) anyway.